source/pug/views/rules-and-regulations.pug

```diff
@@ -2,11 +2,11 @@ include ../partials/header.pug
 
 .nsf-page
   .rules-and-regulations.container
-    .header
-      h3 Wireless Innovation
+    h1.text-center 
```

Comment:
This should actually be the `<h1>` that's used across the site.

Don't break the heading sentence into multiple heading elements for styling purpose. It would sound funky with screen reader and look funky in crawler result.